src/pages/samples/[slug].tsx

```diff
@@ -13,55 +13,126 @@ type PageComponentType = {
-  videoUploadingWebCodecs: dynamic(
-    () => import('../../sample/videoUploadingWebCodecs/main')
-  ),
+};
+
+// Samples that demonstrate functionality specific to WebGPU, or demonstrate the particularities
+// of how WebGPU implements a particular feature within its api. For instance, while many of the
+// sampler parameters in the 'samplerParameters' sample have direct analogues in other graphics api,
+// the primary purpose of 'sampleParameters' is to demonstrate their specific nomenclature and
+// functionality within the context of the WebGPU API.
+const webGPUFeaturesPages: PageComponentType = {
   samplerParameters: dynamic(
     () => import('../../sample/samplerParameters/main')
   ),
-  imageBlur: dynamic(() => import('../../sample/imageBlur/main')),
-  shadowMapping: dynamic(() => import('../../sample/shadowMapping/main')),
   reversedZ: dynamic(() => import('../../sample/reversedZ/main')),
+  renderBundles: dynamic(() => import('../../sample/renderBundles/main')),
+};
+
+// A selection of samples demonstrating various graphics techniques, utilizing various features
+// of the WebGPU API, and often executing render and compute pipelines in tandem to achieve their
+// visual results.
+const graphicsDemoPages: PageComponentType = {
+  cameras: dynamic(() => import('../../sample/cameras/main')),
+  normalMap: dynamic(() => import('../../sample/normalMap/main')),
+  shadowMapping: dynamic(() => import('../../sample/shadowMapping/main')),
   deferredRendering: dynamic(
     () => import('../../sample/deferredRendering/main')
   ),
   particles: dynamic(() => import('../../sample/particles/main')),
+  imageBlur: dynamic(() => import('../../sample/imageBlur/main')),
   cornell: dynamic(() => import('../../sample/cornell/main')),
-  gameOfLife: dynamic(() => import('../../sample/gameOfLife/main')),
-  renderBundles: dynamic(() => import('../../sample/renderBundles/main')),
-  worker: dynamic(() => import('../../sample/worker/main')),
   'A-buffer': dynamic(() => import('../../sample/a-buffer/main')),
-  bitonicSort: dynamic(() => import('../../sample/bitonicSort/main')),
-  normalMap: dynamic(() => import('../../sample/normalMap/main')),
   skinnedMesh: dynamic(() => import('../../sample/skinnedMesh/main')),
 };
 
+// Samples that demonstrate the GPGPU functionality of WebGPU. These samples generally
+// provide a visual representation of the result of a compute operation. Accordingly, the rendering
+// functionality of these samples exists soley to demonstrate the results of these operations.
+// As a general rule of thumb, samples within this category should be of two types:
+//   1. Visualizations of compute operations where the rendering is or could just as effectively be
+//      implemented within canvas2D with minimal changes to the code.
+//   2. Visualizations of compute features that are endemic to the domain of machine learning or A.I
+//      such as 'f16', convolution, backpropogation, etc.
```

Comment:
- f16 is also very useful in graphics
- There are other non-graphics applications of compute like scientific computing, and these too can benefit from ML-focused extensions like dp4a and cooperative matrix multiply.
- It's likely we'll eventually have some samples with text or audio output instead of image output.

I think just point 1 is sufficient to define this category. How about:

```suggestion
// Samples that demonstrate the GPGPU functionality of WebGPU. These samples generally provide some
// user-facing representation (e.g. image, text, or audio) of the result of compute operations.
// Any rendering code is primarily for visualization, not key to the unique part of the sample;
// rendering could also be done using canvas2D without detracting from the sample's usefulness.
```